### PR TITLE
[JA] Translate fledge-api and storage-partitioning 

### PR DESCRIPTION
**Pull request by ilyaspiridonov**
[See review request for index.md on GitLocalize](https://gitlocalize.com/repo/7832/ja/review/50919)
[See review request for index.md on GitLocalize](https://gitlocalize.com/repo/7832/ja/review/50920)
PEMT by Alconost

### CODE REVIEW

**Comment by ilyaspiridonov** [comment]
CC: @agektmr

**Review by agektmr — CHANGES_REQUESTED** [review]

File: site/ja/docs/privacy-sandbox/storage-partitioning/index.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+2. `chrome://flags/#third-party-storage-partitioning` にアクセスします。
+3. 「Experimental Third-party Storage Partitioning」フラグを有効にします。
+
+初期のテストに参加して、安定版のリリース前に Chrome チームが予期しない動作を特定して修正できるように[バグを報告](https://bugs.chromium.org/p/chromium/issues/entry?labels=StoragePartitioning-trial-bugs&components=Blink%3EStorage)してください。Blob URL と Clear-Site-Data ヘッダー API は開発中であり、まだテストには使用できません。
+
+## 更新された API
+
+### ストレージ API
+
+[クォータ システム](https://web.dev/storage-for-the-web/#how-much): クォータ システムは、ストレージに割り当てられるディスク容量を決定するために使用されます。各パーティションはクォータ システムによって個別のバケットとして管理され、許可される容量と、クリアされる時期が決定されます。: `navigator.storage.estimate()` はパーティションの情報を返します。`window.webkitStorageInfo` や `navigator.webkitTemporaryStorage` などの Chrome 専用 API は使用廃止になります。: [IndexedDB](https://developer.mozilla.org/docs/Web/API/IndexedDB_API) と [キャッシュ ストレージ](https://web.dev/cache-api-quick-guide)は、新しいパーティション クォータ システムを使用します。
```
Comment (agektmr) [inline site/ja/docs/privacy-sandbox/storage-partitioning/index.md:45]:
```suggestion
[クォータ システム](https://web.dev/storage-for-the-web/#how-much)
:   クォータ システムは、ストレージに割り当てられるディスク容量を決定するために使用されます。各パーティションはクォータ システムによって個別のバケットとして管理され、許可される容量と、クリアされる時期が決定されます。
:   `navigator.storage.estimate()` はパーティションの情報を返します。`window.webkitStorageInfo` や `navigator.webkitTemporaryStorage` などの Chrome 専用 API は使用廃止になります。
:   [IndexedDB](https://developer.mozilla.org/docs/Web/API/IndexedDB_API) と [キャッシュ ストレージ](https://web.dev/cache-api-quick-guide)は、新しいパーティション クォータ システムを使用します。
```

File: site/ja/docs/privacy-sandbox/storage-partitioning/index.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+[Web Storage API](https://developer.mozilla.org/docs/Web/API/Web_Storage_API) : Web Storage API は、ブラウザがキーと値のペアを保存できる [Local Storage](https://developer.mozilla.org/docs/Web/API/Window/localStorage) と [Session Storage](https://developer.mozilla.org/docs/Web/API/Window/sessionStorage) の 2 つの仕組みを提供します。これらは現在クォータで管理されていませんが、引き続きパーティション化されます。
```
Comment (agektmr) [inline site/ja/docs/privacy-sandbox/storage-partitioning/index.md:47]:
```suggestion
[Web Storage API](https://developer.mozilla.org/docs/Web/API/Web_Storage_API)
:   Web Storage API は、ブラウザがキーと値のペアを保存できる [Local Storage](https://developer.mozilla.org/docs/Web/API/Window/localStorage) と [Session Storage](https://developer.mozilla.org/docs/Web/API/Window/sessionStorage) の 2 つの仕組みを提供します。これらは現在クォータで管理されていませんが、引き続きパーティション化されます。
```

File: site/ja/docs/privacy-sandbox/storage-partitioning/index.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+
+[Origin Private File System](https://web.dev/file-system-access/#accessing-the-origin-private-file-system) : [File System Access API](https://web.dev/file-system-access)を使用すると、ユーザーがアクセスを許可した後、サイトはデバイス上のファイルやフォルダの変更を直接読み取ったり保存したりできます。Origin Private File System を使用すると、オリジンは、ユーザーが簡単にアクセスできるディスクに非公開コンテンツを保存でき、パーティション化されます。
```
Comment (agektmr) [inline site/ja/docs/privacy-sandbox/storage-partitioning/index.md:49]:
```suggestion
[Origin Private File System](https://web.dev/file-system-access/#accessing-the-origin-private-file-system)
:   [File System Access API](https://web.dev/file-system-access)を使用すると、ユーザーがアクセスを許可した後、サイトはデバイス上のファイルやフォルダの変更を直接読み取ったり保存したりできます。Origin Private File System を使用すると、オリジンは、ユーザーが簡単にアクセスできるディスクに非公開コンテンツを保存でき、パーティション化されます。
```

File: site/ja/docs/privacy-sandbox/storage-partitioning/index.md [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+
+[Storage Bucket API](https://wicg.github.io/storage-buckets/explainer.html) : Storage Bucket API は、バケットと呼ばれる新しい概念を使用して、IndexedDB や localStorage などのさまざまなストレージ API を統合する [Storage Standard](https://storage.spec.whatwg.org/) 用に開発されています。バケットに格納されたデータとバケットに関連付けられたメタデータはパーティションされます。
```
Comment (agektmr) [inline site/ja/docs/privacy-sandbox/storage-partitioning/index.md:51]:
```suggestion
[Storage Bucket API](https://wicg.github.io/storage-buckets/explainer.html)
:   Storage Bucket API は、バケットと呼ばれる新しい概念を使用して、IndexedDB や localStorage などのさまざまなストレージ API を統合する [Storage Standard](https://storage.spec.whatwg.org/) 用に開発されています。バケットに格納されたデータとバケットに関連付けられたメタデータはパーティションされます。
```

File: site/ja/docs/privacy-sandbox/storage-partitioning/index.md [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+
+[Clear-Site-Data ヘッダー](https://developer.mozilla.org/docs/Web/HTTP/Headers/Clear-Site-Data) : 応答に `Clear-Site-Data` ヘッダーを含めると、サーバーはユーザーのブラウザに保存されているデータを消去するように要求できます。キャッシュ、Cookie、および DOM ストレージをクリアできます。ヘッダーを使用すると、1 つのパーティション内のストレージのみがクリアされます。
```
Comment (agektmr) [inline site/ja/docs/privacy-sandbox/storage-partitioning/index.md:53]:
```suggestion
[Clear-Site-Data ヘッダー](https://developer.mozilla.org/docs/Web/HTTP/Headers/Clear-Site-Data)
:   応答に `Clear-Site-Data` ヘッダーを含めると、サーバーはユーザーのブラウザに保存されているデータを消去するように要求できます。キャッシュ、Cookie、および DOM ストレージをクリアできます。ヘッダーを使用すると、1 つのパーティション内のストレージのみがクリアされます。
```

File: site/ja/docs/privacy-sandbox/storage-partitioning/index.md [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+
+[Blob URL](https://developer.mozilla.org/docs/Web/API/URL/createObjectURL) ストア : [blob](https://developer.mozilla.org/docs/Web/API/Blob) は、処理される生データを含むオブジェクトであり、リソースにアクセスするために blob URL を生成できます。トップレベル コンテキストで任意の Blob URL に移動するユースケースをサポートするために（[ディスカッション](https://github.com/w3c/FileAPI/issues/153)）、Blob URL ストアはトップレベル サイトではなくエージェント クラスターによってパーティションされます。この機能はまだテストできません。また、パーティションの仕組みが今後変更される可能性があります。
```
Comment (agektmr) [inline site/ja/docs/privacy-sandbox/storage-partitioning/index.md:55]:
```suggestion
[Blob URL](https://developer.mozilla.org/docs/Web/API/URL/createObjectURL) ストア
:   [blob](https://developer.mozilla.org/docs/Web/API/Blob) は、処理される生データを含むオブジェクトであり、リソースにアクセスするために blob URL を生成できます。トップレベル コンテキストで任意の Blob URL に移動するユースケースをサポートするために（[ディスカッション](https://github.com/w3c/FileAPI/issues/153)）、Blob URL ストアはトップレベル サイトではなくエージェント クラスターによってパーティションされます。この機能はまだテストできません。また、パーティションの仕組みが今後変更される可能性があります。
```

File: site/ja/docs/privacy-sandbox/storage-partitioning/index.md [hunk 7]
```diff
@@ -0,0 +1,84 @@
...
+
+### 通信 API
+
+ストレージ API とともに、1 つのコンテキストがオリジンの境界を越えて通信できるようにする通信 API もパーティションされます。この変更は主に、ブロードキャストまたは同一オリジン ランデブーを介して他のコンテキストを検出できるようにする API に影響します。
+
+以下の通信 API については、サードパーティの iframe は同一オリジン コンテキストと通信できなくなります。
+
+[Broadcast Channel](/blog/broadcastchannel/) : Broadcast Channel API を使用すると、[ブラウジング コンテキスト](https://developer.mozilla.org/docs/Glossary/Browsing_context)（ウィンドウ、タブ、または iframe）と同じオリジンのワーカー間の通信が可能になります。: コンテキスト間の関係が明確に定義されているクロスサイト iframe の `postMessage()` は、変更が提案されていません。
```
Comment (agektmr) [inline site/ja/docs/privacy-sandbox/storage-partitioning/index.md:63]:
```suggestion
[Broadcast Channel](/blog/broadcastchannel/)
:   Broadcast Channel API を使用すると、[ブラウジング コンテキスト](https://developer.mozilla.org/docs/Glossary/Browsing_context)（ウィンドウ、タブ、または iframe）と同じオリジンのワーカー間の通信が可能になります。
:   コンテキスト間の関係が明確に定義されているクロスサイト iframe の `postMessage()` は、変更が提案されていません。
```

File: site/ja/docs/privacy-sandbox/storage-partitioning/index.md [hunk 8]
```diff
@@ -0,0 +1,84 @@
...
+
+[SharedWorker](https://developer.mozilla.org/docs/Web/API/SharedWorker) : SharedWorker API は、同一オリジンのブラウジング コンテキスト間でアクセスできるワーカーを提供します。
```
Comment (agektmr) [inline site/ja/docs/privacy-sandbox/storage-partitioning/index.md:65]:
```suggestion
[SharedWorker](https://developer.mozilla.org/docs/Web/API/SharedWorker)
:   SharedWorker API は、同一オリジンのブラウジング コンテキスト間でアクセスできるワーカーを提供します。
```

File: site/ja/docs/privacy-sandbox/storage-partitioning/index.md [hunk 9]
```diff
@@ -0,0 +1,84 @@
...
+
+[Web Locks](https://developer.mozilla.org/docs/Web/API/Web_Locks_API) : Web Locks API を使用すると、同一オリジンの 1 つのタブまたはワーカーで実行されているコードが、何らかの作業の実行中に共有リソースのロックを取得できます。
```
Comment (agektmr) [inline site/ja/docs/privacy-sandbox/storage-partitioning/index.md:67]:
```suggestion
[Web Locks](https://developer.mozilla.org/docs/Web/API/Web_Locks_API)
:   Web Locks API を使用すると、同一オリジンの 1 つのタブまたはワーカーで実行されているコードが、何らかの作業の実行中に共有リソースのロックを取得できます。
```

**Review by agektmr — CHANGES_REQUESTED** [review]

**Comment by ilyaspiridonov** [comment]
> It would be nice if GitLocalize support [definition lists](https://kramdown.gettalong.org/syntax.html#definition-lists)

Hmm, interesting. We'll take a look into this, thanks!

**Review by agektmr — APPROVED** [review]
LGTM

**Pull request merged**